wildfly-openssl.yaml

```diff
@@ -1,10 +1,13 @@
 package:
   name: wildfly-openssl
   version: 2.2.5
-  epoch: 1
+  epoch: 2
   description: OpenSSL-based Java Secure Socket Extension implementation
   copyright:
     - license: Apache-2.0
+  dependencies:
+    runtime:
+      - so:libssl.so.3
```

Comment:
Actually, I think this needs to be `libssl3`

Reply:
Fixed!